ui/src/playlist/PlaylistSongs.js

```diff
@@ -162,6 +163,7 @@ const PlaylistSongs = ({ playlistId, readOnly, ...props }) => {
               {isDesktop && <AlbumLinkField source="album" />}
               {isDesktop && <TextField source="artist" />}
               <DurationField source="duration" className={classes.draggable} />
+              <QualityInfo label="Info" />
```

Comment:
I think the `label` should be `Quality`. Also you are not setting the label in other places.